core/lib/config/src/testonly.rs

```diff
@@ -498,7 +496,6 @@ impl RandomConfig for configs::eth_sender::GasAdjusterConfig {
             internal_enforced_l1_gas_price: g.gen(),
             poll_period: g.gen(),
             max_l1_gas_price: g.gen(),
-            // TODO: this should depend on the mode (Validium or Rollup), but the tests are not adapted yet for this.
```

Comment:
This TODO was removed because this field will be removed in https://github.com/matter-labs/zksync-era/pull/1238